bayesian/api_v1.py

```diff
@@ -248,6 +248,7 @@ class ComponentAnalyses(ResourceWithSchema):
     def get(self, ecosystem, package, version):
         if ecosystem == 'maven':
```

Comment:
Yes, that is what I suggested in https://github.com/fabric8-analytics/fabric8-analytics-worker/pull/23#issuecomment-303336495